backend/src/controllers/AuthController.ts

```diff
@@ -108,40 +136,20 @@ export class AuthController {
       infoSession,
       courseRequirement,
       newsletter,
-      role,
     };
-
-    //check if email already exists
-    const appUserFromEmail: AppUser = await this.appUserService.getAppUserByEmail(email);
-    if (appUserFromEmail != undefined) {
-      console.log(email + ' already has an account');
-      return undefined;
-    }
-
-    //create new app user
-    const newAppUser = await this.appUserMapper.requestToNewEntity(inducteePostRequest); //create new AppUser
-
-    const savedAppUser: AppUser = await this.appUserService.saveAppUser(newAppUser);
-
-    const appUserID: number = savedAppUser.id;
-
-    //create new Firebase
-    const newAccountID: string = await this.accountService.createNewAccount(
-      appUserID,
-      email,
-      password
+    const appUserToSave: AppUser = await this.appUserMapper.requestToExistingEntity(
+      appUserInfo,
+      appUserID
     );
 
-    if (newAccountID === undefined) {
-      return undefined;
-    }
-
+    const savedAppUser: AppUser = await this.appUserService.saveAppUser(appUserToSave);
     return this.appUserMapper.entityToResponse(savedAppUser);
-  }
+  } */
```

Comment:
why is it commented not deleted?

Reply:
Usually I would delete unused code, but in this case I chose to keep them in commented form. This is because we might need the old sign up workflow in the future for whatever reason, maybe this new signup workflow doesn't work as well as we thought it would so we want to switch it back. After a while, when we're sure that we will stick with the new signup workflow, we can just remove the old workflow completely